multi/multi.go

```diff
@@ -454,3 +468,17 @@ func (connMulti *ConnectionMulti) Call17Async(functionName string, args interfac
 func (connMulti *ConnectionMulti) EvalAsync(expr string, args interface{}) *tarantool.Future {
 	return connMulti.getCurrentConnection().EvalAsync(expr, args)
 }
+
+// ExecuteAsync passes a sql expression for execution.
+//
+// Since 1.6.0
```

Comment:
Release 1.6.0 was 15 days ago, so you'll need to pick the next tag (I suppose it'll be 1.7.0) [2]

Reply:
Fixed.